core/kazoo_amqp/src/gen_listener.erl

```diff
@@ -390,6 +395,10 @@ rm_binding(Srv, Binding, Props) ->
 federated_event(Srv, JObj, BasicDeliver, BasicData) ->
     gen_server:cast(Srv, {'federated_event', JObj, BasicDeliver, BasicData}).
 
+-spec the_federator_lives(pid(), {kz_term:ne_binary(), pid()}) -> 'ok'.
+the_federator_lives(Srv, {_Broker, _Pid}=Child) ->
```

Comment:
I would go with something more boring and to the point such as "federator_started" or "notify_federator_started"

Reply:
Where's your sense of whimsy???